modules/rosa-sts-creating-roles.adoc

```diff
@@ -0,0 +1,624 @@
+[id="rosa-sts-creating-roles_{context}"]
+= Creating IAM roles and policies for STS in ROSA
+
+Complete the following steps to set up IAM and OIDC access-based roles before creating your cluster.
+
+.Prerequisites
+
+* Review and complete the deployment prerequisites and policies.
+* Complete setting up the environment for STS.
+
+== Installer access role and policy
+This role is used by the {product-title} (ROSA) CLI to manage installation and deletion of ROSA clusters.
+
+.Procedure
+. Write the trust policy for the `ManagedOpenShift-IAM-Role` installer access role to the [filename]`ManagedOpenShift_IAM_Role.json` file.
++
+.Command to write the trust policy
+[%collapsible]
+====
+[source,terminal]
+----
+$ cat << EOM > ManagedOpenShift_IAM_Role.json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+          "AWS": [
+              "arn:aws:iam::710019948333:role/RH-Managed-OpenShift-Installer"
+          ]
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOM
+----
+====
+. Create the `ManagedOpenShift-IAM-Role` role with the trust policy using the AWS CLI.
++
+[source,terminal]
+----
+$ aws iam create-role --role-name ManagedOpenShift-IAM-Role --assume-role-policy-document file://ManagedOpenShift_IAM_Role_Policy.json [--permissions-boundary ${permissions_boundary_arn}]
```

Comment:
The above filed created is ManagedOpenShift_IAM_Role.json. But the file the commands used is file://ManagedOpenShift_IAM_Role_Policy.json
User cannot run the command directly